doc/sphinx/source/conf.py

```diff
@@ -32,8 +32,15 @@
 # -- Options for HTML output -------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
-html_theme = "alabaster"
-html_static_path = []  # type: ignore
+# use neo4j theme, which extends neo4j docs css for sphinx
+
+html_theme = "neo4j"
+html_theme_path = ["themes"]
+
+# 01-nav.js is a copy of a js file of the same name that is included in the docs-ui bundle
+def setup(app):
+    app.add_js_file('js/01-nav.js', loading_method="defer")
```

Comment:
Is `01-nav.js` a file that changes often?

Reply:
Not often at all. Both these js files are copies of the same files that the docs ui bundles as part of its overall js. The two added here contain the only code needed by this theme, but looking atit again, we might be better off just referencing the already published js: 

```
app.add_js_file("https://neo4j.com/docs/assets/js/site.js", loading_method="defer")
```

Reply:
I'll try this change and just push it if it works.